code/game/machinery/computer/cloning.dm

```diff
@@ -392,6 +395,19 @@
 		SSnanoui.update_uis(src)
 		return
 
+	for(var/obj/machinery/clonepod/pod in pods)
+		if(pod.occupant)
+			if(pod.occupant.ckey == null) // Uses soulhooks
+				for(var/datum/soullink/soulhook/hook in pod.sharedSoulhooks) //Check the soul hooks to see if the to be cloned person is in here
+					if(hook.soulowner.ckey == subject.ckey)
+						scantemp = "Subject already getting cloned."
+						SSnanoui.update_uis(src)
+						return
+			else if(pod.occupant.ckey == subject.ckey) // Directly put in body
```

Comment:
I don't think multiple mobs can can have the same ckey.

Reply:
Glitch in the matrix.

Reply:
True. This part first checked the records. Now it can be removed. Ill do this later tonight